easybuild/framework/easyblock.py

```diff
@@ -2332,6 +2309,44 @@ def extensions_step(self, fetch=False, install=True):
 
             self.ext_instances.append(inst)
 
+    def extensions_step(self, fetch=False, install=True):
+        """
+        After make install, run this.
+        - only if variable len(exts_list) > 0
+        - optionally: load module that was just created using temp module file
+        - find source for extensions, in 'extensions' (and 'packages' for legacy reasons)
+        - run extra_extensions
+        """
+        if not self.cfg.get_ref('exts_list'):
+            self.log.debug("No extensions in exts_list")
+            return
+
+        # load fake module
+        fake_mod_data = None
+        if install and not self.dry_run:
+
+            # load modules for build dependencies as extra modules
+            build_dep_mods = [dep['short_mod_name'] for dep in self.cfg.dependencies(build_only=True)]
+
+            fake_mod_data = self.load_fake_module(purge=True, extra_modules=build_dep_mods)
+
+        self.prepare_for_extensions()
+
+        if fetch:
+            self.exts = self.fetch_extension_sources()
+
+        self.exts_all = self.exts[:]  # retain a copy of all extensions, regardless of filtering/skipping
+
+        # actually install extensions
+        self.log.info("Installing extensions")
```

Comment:
I guess this should be guarded?
```suggestion
        if install:
            self.log.info("Installing extensions")
```